docs/deployment/README.md

```diff
@@ -10,9 +10,12 @@ hide_title: true
 
 This is a guide for deploying a new instance of Cumulus.
 
-**Note:** Cumulus is in the process of moving from a monolithic deployment to
-a more modular deployment process. This document describes the current process.
-Instructions for deploying Cumulus components can be found [here](deployment/components.md).
+This document assumes familiarity with Terraform. If you are not comfortable
+working with Terraform, the following links should bring you up to speed:
+
+* [Introduction to Terraform](https://www.terraform.io/intro/index.html)
+* [Getting Started with Terraform and AWS](https://learn.hashicorp.com/terraform/?track=getting-started#getting-started)
+* [Terraform Configuration Language](https://www.terraform.io/docs/configuration/index.html)
 
 This deployment document is current for the following component versions:
```

Comment:
Not that this is your PR, but what's the point of this paragraph?    The links don't reference a release/ref SHA 🤔